comfy_cli/cql/engine.py

```
@@ -0,0 +1,1513 @@
+    # --- Nested form: descend the subgraph path and write the interior widget. ---
+    if len(segments) > 1:
+        root = next(
+            (n for n in (workflow.get("nodes") or []) if isinstance(n, dict) and str(n.get("id", "")) == segments[0]),
+            None,
+        )
+        if root is None:
+            raise ValueError(f"node {segments[0]} not found in workflow")
+        # Fork a shared definition before mutating so sibling instances stay independent.
+        _isolate_shared_subgraph(workflow, root, defs_by_id)
+        defs_by_id = _subgraph_defs_by_id(workflow)  # rebuild: root.type may have changed
+        # TODO: deep-nest isolation — only the first-hop definition is forked here;
```

Pylint (4.0.5)

[warning] TODO: deep-nest isolation — only the first-hop definition is forked here;

(W0511)